src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -198,7 +198,7 @@ namespace AirLoopHVACDOAS {
         int MixerNum = -1;
         for (auto &dSpec : state.dataAirLoopHVACDOAS->airloopMixer) {
             ++MixerNum;
-            if (UtilityRoutines::SameString(dSpec.name, objectName) && dSpec.m_AirLoopMixer_Num == object_num) {
+            if (Util::SameString(dSpec.name, objectName) && dSpec.m_AirLoopMixer_Num == object_num) {
```

Comment:
First syntactic change: this is probably about 50% of all line changes. Maybe more.